.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        include:
+          - os: ubuntu-latest
+            arch: x86_64
+          - os: windows-latest
+            arch: AMD64
+          - os: macos-latest
+            arch: arm64
```

Comment:
This is potentially problematic. `latest` can change architecture over time; namely macos-latest has recently changed from x86_64 to arm64. I think it's safer to pin the exact VMs.

Reply:
See matching workflow in h5py:
```yaml
          - os: ubuntu-latest
            arch: x86_64
          - os: ubuntu-24.04-arm
            arch: aarch64
          - os: windows-latest
            arch: AMD64
          - os: macos-14
            arch: arm64
          - os: macos-13
            arch: x86_64
```

Reply:
> latest can change architecture over time; namely macos-latest has recently changed from x86_64 to arm64. I think it's safer to pin the exact VMs.

Good point. I'll specify the ubuntu arch explicitly.